### DISCUSSION TITLE
Bee spawning true/false

### DISCUSSION THREAD

**Discussion by BigAl607** [Resourceful-Bees/ResourcefulBees#222] [Suggestions]
I was wondering if it would be possible to add something to the configs that you could turn off or on that would either allow or disallow for the bees to spawn. I'm working on a SkyBlock pack and was the players to create the bees themselves, not get bees through natural spawning

**Comment by Epic428** [comment]
https://wiki.resourcefulbees.com/en/1.16.3/bee_data/spawn_data/

**Reply by BigAl607** [reply]
I went through and changed all the files to change "canSpawnInWorld": true, to false, saved the files, then loaded up the world. The first time I did this, something must have gone wrong, because when I went back to doublecheck, they were all set to true again. Doing it again to see if it did, in fact, save the changes this time.

**Comment by Epic428** [comment]
If you are trying to modify the default bees you need to set generate defaults to false in our common.toml Config file. Otherwise we will overwrite any changes you make to the bees.

**Reply by BigAl607** [reply]
OK. This is why a line that says "Spawn bees? True/False" would work much better. Thanks for your help.

**Reply by Epic428** [reply]
You’re welcome. Though I don’t particularly see how a Config option like that would be any more useful than what we have currently.
